nservicebus/dotnet-templates.md

```diff
@@ -87,6 +87,9 @@ Default: `net462`
 Supported: `net47`, `net462`, `net452`
 
 
+partial: dockerendpoint
```

Comment:
Should the name of this partial be changed to match the new name of the template?